# Service to Set Joints

bool simultaneous
---
int64 ret
string msg
